packages/config/config/devices/0x0098/ct101.json

```diff
@@ -103,42 +177,67 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "0.5°",
+					"label": "0.5° F",
 					"value": 1
 				},
 				{
-					"label": "1.0°",
+					"label": "1.0° F",
 					"value": 2
 				},
 				{
-					"label": "1.5°",
+					"label": "1.5° F",
 					"value": 3
 				},
 				{
-					"label": "2.0°",
+					"label": "2.0° F",
 					"value": 4
 				},
 				{
-					"label": "2.5°",
+					"label": "2.5° F",
 					"value": 5
 				},
 				{
-					"label": "3.0°",
+					"label": "3.0° F",
 					"value": 6
 				},
 				{
-					"label": "3.5°",
+					"label": "3.5° F",
 					"value": 7
 				},
 				{
-					"label": "4.0°",
+					"label": "4.0° F",
 					"value": 8
 				}
 			]
```

Comment:
This label and description don't make sense. What does this do?

Reply:
If this is for param 8, I'm honestly not sure. The documentation does not describe the parameter in detail. This was copied from the `ct100_plus.json` file.

Reply:
It sounds like this is the hysteresis around the setpoint in which the device does not switch on or off. Might be worth calling it "Temperature Hysteresis" instead